packages/web3-eth-personal/src/index.ts

```diff
@@ -0,0 +1,113 @@
+import { Web3Context } from 'web3-core';
+import {
+	Address,
+	HexString,
+	isHexStrict,
+	toChecksumAddress,
+	utf8ToHex,
+	ValidTypes,
+} from 'web3-utils';
+import { Transaction, formatTransaction } from 'web3-eth';
+import { validator } from 'web3-validator';
+import { EthPersonalAPI } from './personal_api';
+
+export class EthPersonal extends Web3Context<EthPersonalAPI> {
+	public async getAccounts() {
+		const result = await this.requestManager.send({
```

Comment:
I think web3.js shouldn't block these functions using await and it should return promise , and let user wait for promise resolution . so user code is able to leverage async based on his requirements. Same suggestion for all following functions.

Reply:
User of the library will be interacting with `getAccounts` function that will be returning the promise in any case. So either we return Promise chain with `then` block to process result or use `await` both have exactly same user experience. 

Use will be able to leverage `async` behavior, no matter we use `await` in the function or not, because the function user is interacting with is returning the Promise. That's the fundamental of adoption of async/await pattern.